src/libANGLE/renderer/d3d/d3d11/SwapChain11.cpp

```diff
@@ -503,7 +504,7 @@ EGLint SwapChain11::resize(DisplayD3D *displayD3D, EGLint backbufferWidth, EGLin
     }
 
     hr = mSwapChain->ResizeBuffers(desc.BufferCount, backbufferWidth, backbufferHeight,
-                                   getSwapChainNativeFormat(), 0);
+                                   getSwapChainNativeFormat(), desc.Flags);
```

Comment:
Can we run into this when passing an `IDXGISwapChain` as an EGL native window? Why is there a  `mSupportsSwapChainResize = false;` step in `SwapChainNativeWindow::initialize()`?